config-default.yml

```diff
@@ -251,7 +256,7 @@ guild:
         duck_pond:                      637821475327311927
         dev_log:                        680501655111729222
         python_news:    &PYNEWS_WEBHOOK 704381182279942324
-
+        incidents_archive:              720671599790915702
```

Comment:
Should `&PYNEWS_WEBHOOK` be indented further? I didn't want to introduce extra unnecessary diff to the config file, but depending on how you look at it, the alignment is now "broken", although at the same time it's not an issue as everything fits.

Reply:
I think it's fine.